documentation/Templates.md

```diff
@@ -67,6 +67,8 @@ Additionally to the [tags and filters](https://stencil.fuller.li/en/latest/built
 | Node | Description |
 | ---- | ----------- |
 | `{% set my_capture_variable %}`<br>`{% endset %}` | node pair allowing you to capture the rendering of nodes inside it, so you can reuse it later by reusing `{{ my_capture_variable }}`.<br>This is useful if you intend to reuse the same content at multiple places — like the `sceneID` is for the `storyboards-default.stencil` template for example. |
+| `{% func my_block a b c %}`<br>`{{args.a}} - {{args.b}} - {{args.c}}`<br>`{% endfunc %}` | node pair allowing you to define a reusable block of nodes inside (like a function), so you can reuse it later.<br>In contrast to the `set` tag, the capturing for `func` is lazy, it will only be processed when actually called. When processed, the block will provide a variable `args` containing all the parameter values.<br>Similar to the `for` tag, this tag provides it's own context scope when called, so variables inside of it are not available outside of it. |
```

Comment:
I have no preference of calling the node to define `macro` instead of `func`, but I'm not a fan of the Jinja2 syntax for the parameters, also because as @djbe pointed out, we'd have to parse them ourselves while with our current syntax for `func` (1) we use the already existing methods of Stencil to parse node arguments (2) it makes the syntax more consistent with all other Stencil nodes that use spaces to separate arguments of various Nodes.